src/v2/view-builder/views/ov/ChallengeAuthenticatorDataOktaVerifyView.js

```diff
@@ -46,7 +46,12 @@ const Body = SelectAuthenticatorVerifyViewBody.extend({
   }
 });
 
-export default BaseAuthenticatorView.extend({
+const AuthenticatorView = BaseAuthenticatorView.extend({
```

Comment:
seems that we don't really need to create a View here, so maybe we can just export the Body instead?
```
export default Body;
```
and we could even rename this to "ChallengeAuthenticatorDataOktaVerifyBody.js"